thoth/storages/graph/postgres.py

```diff
@@ -3852,7 +3853,12 @@ def create_python_cve_record(
         package_version = self.normalize_python_package_version(package_version)
         with self._session_scope() as session, session.begin(subtransactions=True):
             cve, _ = CVE.get_or_create(
-                session, cve_id=record_id, version_range=version_range, advisory=advisory, cve_name=cve
+                session,
+                cve_id=record_id,
+                version_range=version_range,
+                advisory=advisory,
+                cve_name=cve,
+                aggregated_at=datetime.now(),
```

Comment:
@fridex I wanted to add `datetime.now()` on this line https://github.com/bissenbay/cve-update-job/blob/master/app.py#L94. However, as we did not want to capture the download time I chose this place. However, the only concern I have here is that this will create duplicates of CVE records with different datetime.

Reply:
Yes, that's true. :+1: 

Let's check then whether the database always states uniquely `cve_id` or `cve_name` and try to query based on it.

Reply:
Most of the times `cve_name` is just empty string, but `cve_id` seems to be unique, so what about if we use both `cve_id` and `cve_name` instead of just `cve_id`?

Reply:
If `cve_id` is unique, I think we should be just fine to use it not to complicate things.